docs/specification/current.md

```diff
@@ -554,6 +554,24 @@ The following example illustrates a delete action:
 }
 ```
 
+###### System Action
```

Comment:
I understand explaining this next to suggested actions, but it might be a little confusing being within the card attributes section, though it's not part of a card.  I Might make more sense after the card attributes section (after card links), before the CDS service response example.

Reply:
Dennis - great suggestion, thank you. Moved. Also, great work on the consistent re-reviews over the past few days.